app/javascript/plugins/progress-tracker.js

```diff
@@ -0,0 +1,79 @@
+// @flow
+import $ from 'jquery';
+import ee from '../shared/pub_sub';
+
+export default class ProgressTracker {
+  constructor() {
+    this.addEventListeners();
+  }
+
+  addEventListeners() {
+    ee.on('action:submitted_success', this.show);
+    ee.on('action:submitted_success', () => this.tick('signed'));
+    ee.on('fundraiser:transaction_success', () => this.tick('donated'));
+    window.addEventListener(
+      'share',
+      () => this.addEventListeners.tick('shared'),
+      false
+    );
+    $('.two-step__decline').on('click', () => {
+      this.cross('shared');
+    });
+  }
+
+  show() {
+    const $tracker = $('.header-logo .progress-tracker');
+    $tracker
+      .removeClass('hidden-closed')
+      .clone()
+      .addClass('progress-tracker--fundraiser')
+      .prependTo('.Stepper.fundraiser-bar__top');
+  }
+
+  tick(section: string) {
+    $(`.progress-tracker__circle--${section}`)
+      .removeClass('progress-tracker__circle--cross')
+      .addClass('progress-tracker__circle--tick');
+  }
+
+  cross(section: string) {
+    console.log('cross:', section);
+    $(`.progress-tracker__circle--${section}`)
+      .removeClass('progress-tracker__circle--tick')
+      .addClass('progress-tracker__circle--cross');
+  }
+
+  setup() {
+    ee.on('action:submitted_success', function() {});
+  }
+}
+/*
+window.ee.on('action:submitted_success', function() {
```

Comment:
There's a bunch of commented stuff here - did you mean to remove this?

Reply:
Yup! removed both!